shell/components/nuxt/nuxt-child.js

```diff
@@ -57,66 +43,14 @@ export default {
       }
     }
 
-    // make sure that leave is called asynchronous (fix #5703)
-    if (transition.css === false) {
-      const leave = listeners.leave
-
-      // only add leave listener when user didnt provide one
-      // or when it misses the done argument
-      if (!leave || leave.length < 2) {
-        listeners.leave = (el, done) => {
-          if (leave) {
-            leave.call(_parent, el)
-          }
-
-          _parent.$nextTick(done)
-        }
-      }
-    }
-
     let routerView = h('routerView', data)
 
     if (props.keepAlive) {
       routerView = h('keep-alive', { props: props.keepAliveProps }, [routerView])
     }
 
     return h('transition', {
```

Comment:
Is this still needed?

Reply:
Not really... Can be just a plain `div` afaik.